core-bundle/src/Image/Studio/StudioFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Image\Studio;
+
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class StudioFactory implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    /**
+     * Create a new image studio instance.
+     *
+     * As `contao.image.studio` is a non-shared service you can also directly
+     * use the dependency injection container to get a single new instance.
+     */
+    public function __invoke(): Studio
+    {
+        return $this->container->get('contao.image.studio');
```

Comment:
I don't really like this pattern. Can't we just make the factory itself accessible and force developers to do

```php
$studio = $this->get('contao.image.studio_factory')->create();
```

It's more explicit that you're getting a new instance every time you call it, imho.

Reply:
Ah, but that's already the pattern, isn't it? Do you effectively mean renaming `__invoke()` to `create()`?